examples/channels-example/README.md

```diff
@@ -0,0 +1,111 @@
+
+For RIG we could use default configuration values, but where's fun in that. Let's change them so we can also see how to configure certain things to our needs.
+
+```sh
+# Description of environment variables
+
+# PORT => Port at which we want to expose RIG, by default 4000
+# PRIVILEGED_ROLES => This is a string array of roles that can see also broadcasts from other people, not just it's own, by default empty array
+# JWT_ROLES_FIELD => Key in JWT claims/payload under which PRIVILEGED_ROLES are set for each user, by default roles
+# JWT_USER_FIELD => Key in JWT claims/payload that is used to construct topic name for Phoenix channel, by default user
+# KAFKA_USER_FIELD => Key in Kafka message, by which RIG knows where to broadcast, by default user
+# KAFKA_SOURCE_TOPICS => Name of Kafka topic to which consumer will connect, by default rig
+# JWT_SECRET_KEY => Secret key by which JWTs are signed, by default empty string
+
+PORT=7000 \
+PRIVILEGED_ROLES=admin \
+JWT_ROLES_FIELD=levels \
+JWT_USER_FIELD=username \
+JWT_SECRET_KEY=mysecret \
+KAFKA_USER_FIELD=username \
+KAFKA_SOURCE_TOPICS=example \
+mix phx.server
+```
+
+RIG is now available on port `7000`.
+
+**Terminal 2:** Start Service (from service directory)
+
+```
+# Go to service folder
+npm i
+npm start
+```
+
+Service is now available on port `8000`.
+
+**Terminal 3:** Start Frontend (from frontend directory)
+
+```
+# Go to frontend folder
+npm i
+npm start # Should automatically open browser window with http://localhost:3000
+```
+
+Add new API to RIG. REST API calls from Frontend goes through RIG to external services, thus we need to tell RIG that such API exists (or define it in initial JSON file). At the same time by this we are testing proxy part of RIG.
+
+```
+curl -X "POST" \
+-H "Content-Type: application/json" \
+-d "{\"id\":\"kafka-service\",\"name\":\"kafka-service\",\"version_data\":{\"default\":{\"endpoints\":[{\"id\":\"kafka-producer-endpoint\",\"path\":\"/produce\",\"method\":\"POST\",\"not_secured\":true}]}},\"proxy\":{\"use_env\":false,\"target_url\":\"localhost\",\"port\":8000}}" \
+--silent \
+"http://localhost:7000/apis"
+```
+
+## Example scenarios to test
+
+Here are a few examples you may try with this simple setup.
+
+### Subscribe to my own channel
+
+1. Open two tabs in browser with `http://localhost:3000`
+1. First tab: Select some transport type (doesn't matter which)
+1. First tab: Fill in `User Name` with e.g. mike
+1. First tab: Leave `Levels` field empty
+1. First tab: Fill in `Subscribe to topic` with e.g. mike (this has to be the same as your `User Name`)
+1. First tab: Press `Connect` => You should see spinner and message on the right side, that you are connected (as well as logs in RIG terminal)
+1. Do the same in second browser tab
+1. First tab: Fill in `Event message` with e.g. (again username has to match) `{"username":"mike", "payload":"hello"}` (`username` is the only required field, rest of the message can be whatever)
```

Comment:
perhaps you could mention (here and/or on the UI) that the rest may be anything, but for the UI "payload" has be used, as the UI renders this field only

Reply:
updated to UI to reflect this text